src/api/app/views/webui2/webui/user/_activity.html.haml

```diff
@@ -21,4 +21,12 @@
                 classname = activity_classname(activity, percentiles)
 
               %td{ title: "#{current_day} - #{activity} contributions", class: classname }
-                &nbsp;
+                - if activity > 0
+                  = link_to(user_show_path(user, params: { date: current_day }), class: 'activity-link') do
+                    &nbsp;
```

Comment:
couldn't we use `table { empty-cells: show; }` and remove the `&nbsp` @coolo ?

Reply:
I can try. I wasn't aware of that css

Reply:
works

Reply:
No, I was wrong. Without the &nbsp; you can't click the links. And if I add &nbsp; only to links, it will render the table very imbalanced. Will revert that commit

Reply:
ok, sorry for that.

Reply:
No need to be sorry, I learned a new trick (that is just not applicable here :)